vkcs/internal/util/util.go

```diff
@@ -315,3 +316,23 @@ func ErrorWithRequestID(err error, requestID string) error {
 	}
 	return fmt.Errorf("%w\nRequest ID: %s", err, requestID)
 }
+
+func GetFirstNotEmpty(values ...string) string {
+	for _, value := range values {
+		if len(value) != 0 {
+			return value
+		}
+	}
+
+	return ""
+}
+
+func GetFirstNotEmptyValue(values ...types.String) string {
```

Comment:
It's bad habit to dump all utilities in a single file. We at least have `internal/framework/utils` package where it should be placed.